pkg/utils/secrets/basic_auth.go

```diff
@@ -44,6 +44,7 @@ type BasicAuth struct {
 
 	Username string
 	Password string
+	Auth     []byte
```

Comment:
```suggestion
	auth     []byte
```

Nit: Make field private because it should not be settable from outside.